### PR TITLE
Create generators

### PR DESCRIPTION
**Pull request by TJBANEY**

### CODE REVIEW

**Comment by texas2010** [comment]
@TJBANEY  Please read this [style guide](https://github.com/freeCodeCamp/guides#article-style-guide).

File: src/pages/python/generators [hunk 1]
```diff
@@ -0,0 +1 @@
+A Python generator is a function which returns a generator iterator by calling yield.
```
Comment (bla-ke) [inline src/pages/python/generators]: Add in the first line:
```
---
title: Generators
---
```

**Comment by TJBANEY** [comment]
I will go in and style it correctly, and add more content.

**Comment by systimotic** [comment]
Closing as a duplicate of https://github.com/freeCodeCamp/guides/pull/4724 Happy coding!

**Pull request closed without merging**